src/pages/dar_collection_review/ApplicationInformation.js

```diff
@@ -53,6 +61,26 @@ const generateLabelSpanContents = (labelValue, key,  spanValue, isLoading) => {
   );
 };
 
+// function to generate application details content
+const dynamicRowGeneration = (rowElementMaxCount, appDetailLabels, loading) => {
+  const labelArray = appDetailLabels.map(label => {
+    // only generate elements that are populated
+    if (label[0] && label[0]!='' && label[0].length > 0) {
+      return generateLabelSpanContents(label[1], label[2], label[0], loading);
```

Comment:
I remember you mentioned in a previous comment - Boolean values don't render as text in React (annoying, I know) so you'll have to convert it yourself. Odds are this won't just apply to anvilStorage.

If I was handling different types separately (see above comment) then I could potentially also handle the rendering of text for booleans, right?

Reply:
Yeah you could. Once you know that there's a value you can check to see if you need to do some processing for output. 

Lodash has its [isBoolean](https://lodash.com/docs/4.17.15#isBoolean) method that you can use for booleans. Another option is to use <varName>.toString() and compare values that way. 

NOTE: There's probably even more options, these are just the two I just came up with off the top of my head.